doc/api/process.md

```diff
@@ -852,13 +871,15 @@ And `process.argv`:
 added: v0.1.100
 -->
 
+* {String}
+
 The `process.execPath` property returns the absolute pathname of the executable
 that started the Node.js process.
 
 For example:
 
 ```sh
```

Comment:
This shouldn't be `sh` if we're converting it to a JavaScript string value, it should be `js`.